packages/compass/src/main/auto-updater.ts

```diff
@@ -26,7 +28,29 @@ class LinuxAutoUpdater extends EventEmitter implements AutoUpdater {
   }
 }
 
-const autoUpdater =
-  process.platform === 'linux' ? new LinuxAutoUpdater() : electronAutoUpdater;
+function hasSquirrel() {
+  const updateExe = path.resolve(
+    path.dirname(process.execPath),
+    '..',
+    'Update.exe'
+  );
+  return pathExistsSync(updateExe);
```

Comment:
We currently don't have a dependency on `fs-extra` anywhere except for hadron-build package, so I'd maybe suggest avoiding it, it should be pretty straightforward to just do with using `fs.statSync`